apps/user-profile/src/app/app-routing.module.ts

```diff
@@ -127,6 +127,42 @@ const routes: Routes = [
             path: 'mailingLists',
             component: SettingsMailingListsComponent,
             data: { breadcrumb: 'SETTINGS.MAILING_LISTS' },
+            children: [
+              {
+                path: '',
+                component: SettingsMailingListsComponent,
+                data: { breadcrumb: 'SETTINGS.MAILING_LISTS' },
+              },
+              {
+                path: ':voName',
+                component: SettingsMailingListsComponent,
+                data: { breadcrumb: 'SETTINGS.MAILING_LISTS' },
+                children: [
+                  {
+                    path: '',
+                    component: SettingsMailingListsComponent,
+                    data: { breadcrumb: 'SETTINGS.MAILING_LISTS' },
+                  },
+                  {
+                    path: ':resourceName',
+                    component: SettingsMailingListsComponent,
+                    data: { breadcrumb: 'SETTINGS.MAILING_LISTS' },
+                    children: [
+                      {
+                        path: '',
+                        component: SettingsMailingListsComponent,
+                        data: { breadcrumb: 'SETTINGS.MAILING_LISTS' },
+                      },
+                      {
+                        path: ':subscribe',
+                        component: SettingsMailingListsComponent,
+                        data: { breadcrumb: 'SETTINGS.MAILING_LISTS' },
+                      },
+                    ],
+                  },
+                ],
+              },
+            ],
```

Comment:
In my opinion this should be done by query parameters
from their definition: "extensions of the URL that are utilized to help determine specific content or action based on the data being delivered" 
it seems more proper to use this, so we can avoid this not very nice looking code
but lets discuss it on the daily sprint please